cleanlab/datalab/internal/issue_finder.py

```diff
@@ -358,4 +372,7 @@ def get_available_issue_types(self, **kwargs):
                 )
                 issue_types_copy.pop("outlier")
 
+        if self.datalab.task == "regression":
+            issue_types_copy = {**issue_types_copy, **{"label": {}}}
+
```

Comment:
The problem is in `find_issues` method in `IssueFinder`:
```
    def find_issues(
        self,
        *,
        pred_probs: Optional[np.ndarray] = None,
        features: Optional[npt.NDArray] = None,
        knn_graph: Optional[csr_matrix] = None,
        issue_types: Optional[Dict[str, Any]] = None,
    ) -> None:
        issue_types_copy = self.get_available_issue_types(
            pred_probs=pred_probs,
            features=features,
            knn_graph=knn_graph,
            issue_types=issue_types,
        )
        if not issue_types_copy:
            return None
```
For regression it could be called without arguments `find_issues()`, wihtout this "if" in `get_available_issue_types` result will be `None`. `issue_types_copy = {**issue_types_copy, **{"label": {}}}` ensures that we keep e.g. issue types from  `knn_graph` if provided in issues_types_copy  and adds `label` - for regression we can't dectect it from `pred_probs`. I hope this makes sense, but I could miss something obvious.